src/web/components/Callout/Form.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { calloutCampaign } from '@root/fixtures/calloutCampaign';
+import { Form } from './Form';
+
+export default {
+    component: Form,
+    title: 'Components/Callout/Form',
+};
+
+export const Default = () => {
+    return (
+        <div
+            className={css`
+                width: 630px;
+                padding: 15px;
+            `}
+        >
+            <Form
+                formFields={calloutCampaign.formFields}
+                submitForm={() => {}}
+                formId={1234}
+            />
+        </div>
+    );
+};
+Default.story = { name: 'default' };
+
```

Comment:
It'd be great to see more stories with variations on the form data being passed in

Reply:
That reminds me, I am unsure how to support a form with no fields? (its an edge case that should not arise in theory)

The `calloutCampaign` fixture contain all the types of fields we support. We can split out each field into its own story but I do not see this being particularly beneficial bar have diffs for an exact Field rather than the form